### PR TITLE
Add PagingView to Collection View section

### LINKED ISSUE #781: PagingView
[PagingView](https://github.com/KyoheiG3/PagingView) - Infinite paging, Smart auto layout, Interface of similar to UIKit. :large_orange_diamond:

### PR DESCRIPTION
**Pull request by miguelarauj1o**
Close #781

## Project URL
https://github.com/KyoheiG3/PagingView

## Checklist
- [x] Only one project/change is in this pull request
- [x] Addition in chronological order (bottom of category)
- [x] Supports iOS 9 / tvOS 10 or later
- [x] Supports Swift 3
- [x] Has a commit from less than 2 years ago
- [x] Has a **clear** README in English

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -2117,7 +2117,8 @@ Most of these are paid services, some have free tiers.
 * [SwiftSpreadSheet](https://github.com/stuffrabbit/SwiftSpreadsheet) - Spreadsheet CollectionViewLayout in Swift. Fully customizable. 🔶
 * [GenericDataSource](https://github.com/GenericDataSource/GenericDataSource) - A generic small reusable components for data source implementation for UITableView/UICollectionView in Swift. 🔶
 * [BouncyLayout](https://github.com/roberthein/BouncyLayout) - BouncyLayout is a collection view layout that makes your cells bounce. 🔶
-* [Savory](https://github.com/Nandiin/Savory) - A swift accordion view implementation. 🔶
+* [Savory](https://github.com/Nandiin/Savory) - A swift accordion view implementation.
```
Comment (dkhamsing) [inline README.md]: hey we're missing 🔶  at the end of Savory

Comment (miguelarauj1o) [inline README.md]: ops...

**Comment by dkhamsing** [comment]
thx @miguelarauj1o

**Comment by miguelarauj1o** [comment]
@dkhamsing, please, can you squash and merge this PR?

**Comment by miguelarauj1o** [comment]
Great, thx!

**Comment by miguelarauj1o** [comment]
You're fast! hahah

**Pull request merged**